mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -1,16 +1,22 @@
 CLASS net/minecraft/server/MinecraftServer
-	CLASS class_643 ServerThread
-	CLASS class_645 ConsoleInputThread
```

Comment:
Wait, what's happening here? Did I accidentally unmap these?

Reply:
Enigma moved them to the bottom I think